index.html

```diff
@@ -0,0 +1,175 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <span></span>
+    <link rel="stylesheet" href="static/style.css" />
+    <link
+      rel="stylesheet"
+      href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css"
+      integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk"
+      crossorigin="anonymous"
+    />
```

Comment:
Why do we have both the Bootstrap CSS and the CDN?

Reply:
i thought i got rid of the css